app/views/users/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<section>
+<%= render "user", users:@user%>
+
+<div class="bio-container">
+<h3 class="bio">Bio</h3>
+<p><%= @user.bio%></p>
+<%@user.three_recent_posts.each_with_index do |pt,index|%>
+<%= render 'posts/post', post:pt, index:index %>
+<%end%>
+</div>
+<button><%= link_to 'See All Posts', user_posts_path(@user), class: 'btn' %>
+</button>
+</section>
```

Comment:
- [ ] Good job so far but make sure that your **See all Post** links open user's post index has a design match with the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png). You design is a bit different. Please fix it as expected for this project :+1: